### PR TITLE
Add achievement type constants

### PR DESCRIPTION
**Pull request by luchaos**
- Remove development tutorial game checks

### CODE REVIEW

File: src/AwardType.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ abstract class AwardType
 
     public const PATREON_SUPPORTER = 6;
 
-    public static $active = [
+    public static array $active = [
```
Comment (Jamiras) [inline src/AwardType.php:19]: What's the difference between "static array" here and "const" for AchievementType::FLAGS? Shouldn't they both use the same syntax?

**Review by Jamiras — APPROVED** [review]
I'm guessing there's more places still using 3/5 (the ticket filters page comes to mind), but we can get them in the future.

File: src/AwardType.php (hunk 1 above)
Comment (luchaos) [reply src/AwardType.php:19]: Constants in PHP 8.0 cannot have a type - static properties should.
I was adding the missing type as the IDE complained about it.
Now that you mention it - it should be a const, too, as it's not to be changed during runtime.
I'll change that 👍

Comment (luchaos) [reply src/AwardType.php]: See https://github.com/RetroAchievements/RAWeb/pull/986/commits/902d3b5b2c5216942025d1b4f5f30cb85017bdc2

**Pull request merged**